src/patternfly/demos/PageLayout/page-template-header-tools-elements.hbs

```diff
@@ -16,11 +16,11 @@
         {{/button}}
       {{/toolbar-item}}
       {{#> toolbar-item toolbar-item--modifier="pf-u-sr-only pf-u-visible-on-md"}}
-        {{#> dropdown dropdown-toggle--modifier="pf-m-plain" id="dropdown-example-collapsed" dropdown--attribute="pf-m-plain" dropdown--HasToggleIcon="true"}}
-          Kyle Baker
+        {{#> dropdown id=(concat page--id '-dropdown') dropdown-toggle--modifier="pf-m-plain" dropdown--attribute="pf-m-plain" dropdown--HasToggleIcon="true"}}
+          <i class="fas fa-cog"></i>
```

Comment:
When you removed "Kyle Baker" you also removed the only accessible text string associated with this component. Can you add an `aria-label` to this dropdown?

Also, at sizes larger than 992px, I'm seeing this which looks like a mistake. Do we need an issue for that? 
![image](https://user-images.githubusercontent.com/21063328/49451016-3f1b7600-f7ac-11e8-94ce-2fbe1bb38ae3.png)